pkg/extract/extract_test.go

```diff
@@ -9,21 +9,59 @@ import (
 	"testing"
 
 	"github.com/code-ready/crc/pkg/crc/logging"
+	crcos "github.com/code-ready/crc/pkg/os"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
-type FileMap map[string]string
+type fileMap map[string]string
 
-func copyFileMap(orig FileMap) FileMap {
-	copy := FileMap{}
+var (
+	files fileMap = map[string]string{
+		"/a/b/c.txt": "ccc",
+		"/a/d/e.txt": "eee",
+	}
+	filteredFiles fileMap = map[string]string{
+		"/a/b/c.txt": "ccc",
+	}
+	archives = []string{
+		"test.tar",
+		"test.tar.gz",
+		"test.zip",
+		"test.tar.xz",
+	}
```

Comment:
During this unit test can we also put a `crcbundle` tarball file in the testdata folder just to make sure it actually able to detect it as a tarball?

Reply:
Yes this is exactly the second test https://github.com/code-ready/crc/pull/1428/files#diff-92bf4f1e87f4458b933a72c9b54411d5R42

crcbundle can be any type now and it checks for it.